adapters/stroeerCore/stroeercore.go

```diff
@@ -59,17 +65,35 @@ func (a *adapter) MakeBids(bidRequest *openrtb2.BidRequest, requestData *adapter
 			Price: bid.CPM,
 			AdM:   bid.Ad,
 			CrID:  bid.CrID,
+			MType: markupType,
 		}
 
 		bidderResponse.Bids = append(bidderResponse.Bids, &adapters.TypedBid{
 			Bid:     &openRtbBid,
-			BidType: openrtb_ext.BidTypeBanner,
+			BidType: bidType,
 		})
 	}
 
 	return bidderResponse, errors
 }
 
+func getBidMediaTypeForImp(impID string, imps []openrtb2.Imp) (emptyBidType openrtb_ext.BidType, emptyMarkupType openrtb2.MarkupType, err error) {
+	for _, imp := range imps {
+		if imp.ID == impID {
+			if imp.Banner != nil {
+				return openrtb_ext.BidTypeBanner, openrtb2.MarkupBanner, nil
+			}
+			if imp.Video != nil {
+				return openrtb_ext.BidTypeVideo, openrtb2.MarkupVideo, nil
+			}
+		}
+	}
```

Comment:
Warning - This is an anti-pattern. This code assumes that for multi-format bid request, mediatype will be set to type banner by default. I would recommend to set the mediaType based on bidder response. Your bidder should be able to set openrtb2.Bid.MType field and use that field here to get mediaTypeForImp. Even if your bidder does not support multi-format request, I would highly encourage you to make this change.
Here is an example of how to do it - https://github.com/prebid/prebid-server/blob/ec729e6e5d50a1306d76332fda52108b9d313c1d/adapters/aidem/aidem.go#L89-L102